src/pages/login.svelte

```diff
@@ -12,6 +16,44 @@
     import MediaQuery from '~/components/utils/media-query.svelte'
     import Features from '~/components/elements/features.svelte'
     import UnicoveAnimated from '~/components/elements/unicove-animated.svelte'
+
+    const whalesplainerUrl = import.meta.env.SNOWPACK_PUBLIC_WHALESPLAINER_URL
+
+    let creatingAccount = false
+
+    async function createAccount(event: Event) {
```

Comment:
Errors thrown here are not caught

Reply:
The account creation lib API isn't very consistent. It would be better if it throws an error if the response contains an "error" key. That would also allow the response type to be CallbackPayload avoiding the type inference issues

Reply:
Yeah that would def solve the issue. I'll change that 👍

Reply:
@jnordberg done!

Reply:
Errors thrown in the function are still not handled, a good pattern for this is to wrap it in a non-async function that handles the event and does a .catch

Reply:
This also allows you to move the `creatingAccount` variable out of the async function, setting it to true in that wrapper function and again to false in a .finally chain on the promise